src/config/dynamic-theme-fixes.config

```diff
@@ -567,6 +567,15 @@ INVERT
 
 ================================
 
+blogs.windows.com
+
+CSS
+.c-logo .c-image {
+    background: white !important;
```

Comment:
https://github.com/darkreader/darkreader/pull/3115/files#diff-04c6e90faac2675aa89e2176d2eec7d8R119 I think we better could use the newly introduced variable